test/integration/targets/copy/tasks/setgid.yml

```
@@ -0,0 +1,27 @@
+- block:
+    - name: Create test directory
+      file:
+        path: "{{ remote_tmp_dir }}/test_setgid"
+        state: directory
+        mode: '6750'
```

```suggestion
        mode: '2750'
```

Is there a reason to set both `setuid` and `setgid`? Since we are testing `setgid` here I would set just that.

---

Good point. I had just copied one of the reproducers from the issue.